packages/features/bookings/lib/booking-report.utils.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Extracts the booker email from booking attendees
+ * Convention: first attendee is the booker
+ */
+export function extractBookerEmail(attendees: Array<{ email: string }>): string | null {
+  return attendees[0]?.email || null;
+}
```

Comment:
This function shouldn't care about entire attendees array, that's unnecessary information passing. I'm adding a normalizer for watchlist where we normalize emails and domains, and also perform extraction. I'd say we can re-use that method for a more robust approach.